mdal/frmts/mdal_xmdf.hpp

```diff
@@ -58,6 +58,26 @@ namespace MDAL
   class DriverXmdf: public Driver
   {
     public:
+      /**
+       * Driver for XMDF Files
+       *
+       * Structure of the file. Groups are optional since it depends
+       * on tools which groups are created.
+       * - root
+       *   - Temporal
+       *     - Depth
+       *     - Velocity
+       *     - ..
+       *   - Maximums
+       *     - Depth
+       *     - Velocity
+       *     - ..
+       *   - Difference (res_to_res.exe TUFLOW utility tool)
+       *     - ..
+       *   - Times (e.g. time of peak velocity)
+       *     - ..
+       *   - ...
```

Comment:
Maybe worth saying that this is the structure used by TUFLOW but other solvers may use a different structure within XMDF?